src/app/onion/learning-object-builder/components/column-wrapper/column-wrapper.component.html

```diff
@@ -1,8 +1,10 @@
 <div class="builder-template-columns">
   <div #columnWrapper class="columns" [ngStyle]="{'top': columnOffset + 'px'}" [ngClass]="{'no-right': !rightColumn, 'no-left': !leftColumn}">
-    <div *ngIf="showColumn('l')" class="left-column column">
-      <div class="content">
-        <ng-content select="[left]"></ng-content>
+    <div class="scroll-wrapper" [ngStyle]="{'height': columnHeight + 'px'}">
```

Comment:
How does this look on the outcomes and materials page? I think that the column is smaller on those pages, but I could be wrong.

Reply:
It's the same size but if the width is small the components looks shrinked, this is for both the info section and outcome.  Does not break the UI though.

Normal:
![Screen Shot 2019-03-27 at 2 09 39 PM](https://user-images.githubusercontent.com/32078831/55101239-0f191b00-509a-11e9-86f3-37b266b6b058.png)

Shrinked:
![Screen Shot 2019-03-27 at 2 09 54 PM](https://user-images.githubusercontent.com/32078831/55101250-13453880-509a-11e9-8430-b9879216f928.png)